### PR TITLE
webpack react start project

### PR DESCRIPTION
**Pull request by chrxs**

### CODE REVIEW

File: webpack-react/config/phantomjs-shim.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+(function () {
+
+  var Ap = Array.prototype;
+  var slice = Ap.slice;
+  var Fp = Function.prototype;
+
```
Comment (jacksonhyde) [inline webpack-react/config/phantomjs-shim.js]: Shouldn't `Ap` and `Fp` be camel case?

Comment (jacksonhyde) [inline webpack-react/config/phantomjs-shim.js]: Or are they class constructors?

Comment (cajoy) [inline webpack-react/config/phantomjs-shim.js]: It is not really good behaviour when you use abbreviations for variables.

the best way to describe is: arrayPrototype or functionPrototype if it is needed.

File: webpack-react/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+[NVM](https://github.com/creationix/nvm) to manage node versions on your machine.
+
+Run `npm install` to install all dependancies listed in `package.json`
+
+## Running the dev-server
+The project uses webpack and webpack-dev-server for development environment, a script tasks exists in the `package.json` to start this, from the terminal run:
+`npm start` and load `http://localhost:8080` in your browser.
+Making changes should hot reload JS files and reload CSS files without the need to refresh the browser.
+
+## Building babylon webapp
```
Comment (jacksonhyde) [inline webpack-react/README.md]: babylon? :)

File: webpack-react/config/phantomjs-shim.js (hunk 1 above)
Comment (jacksonhyde) [inline webpack-react/config/phantomjs-shim.js]: +1

Comment (chrxs) [inline webpack-react/config/phantomjs-shim.js]: I actually just copied this (and a lot of the rest of this commit) from [https://github.com/bradleyboy/yarsk]

File: webpack-react/README.md (hunk 2 above)
Comment (chrxs) [inline webpack-react/README.md]: ha ha whooops, was gonna happen somewhere

**Comment by jacksonhyde** [comment]
+R

**Pull request merged**